easypost/__init__.py

```diff
@@ -531,19 +482,16 @@ def refresh(self):
         return self
 
     @classmethod
-    def class_name(cls):
-        if cls == Resource:
-            raise NotImplementedError('Resource is an abstract class. '
-                                      'You should perform actions on its subclasses.')
-
-        cls_name = six.text_type(cls.__name__)
-        cls_name = cls_name[0:1] + re.sub(r'([A-Z])', r'_\1', cls_name[1:])
-        return "%s" % cls_name.lower()
+    def snakecase_name(cls):
+        """ Return the class name as snake_case.
```

Comment:
Nit: Let's kill the extra space between `" Return`.

One-liner docstrings should also be on a single line.

https://www.python.org/dev/peps/pep-0257/